source/grid/grid_tools.cc

```diff
@@ -6938,30 +6938,22 @@ namespace GridTools
             if (std::abs(iso_level - ls_values[mask[0]]) < tolerance)
               vertices.emplace_back(points[mask[0]]);
             else if (std::abs(iso_level - ls_values[mask[1]]) < tolerance)
-              {
-                if (i + 1 == n_subdivisions)
-                  vertices.emplace_back(points[mask[1]]);
-              }
-            // check for a saddle point, if iso_level values of the two
-            // corner nodes are identical
-            else if ((std::abs(ls_values[mask[0]] - ls_values[mask[1]]) <
-                      tolerance) &&
-                     (ls_values[mask[0]] >= iso_level))
-              vertices.emplace_back(0.5 * (points[mask[0]] + points[mask[1]]));
-            // check if the edge is cut
-            else if (((ls_values[mask[0]] > iso_level) &&
-                      (ls_values[mask[1]] < iso_level)) ||
-                     ((ls_values[mask[0]] < iso_level) &&
-                      (ls_values[mask[1]] > iso_level)))
-              {
-                // determine the interpolation weight (0<mu<1)
-                const double mu = (iso_level - ls_values[mask[0]]) /
-                                  (ls_values[mask[1]] - ls_values[mask[0]]);
+              if (i + 1 == n_subdivisions)
+                vertices.emplace_back(points[mask[1]]);
+              // check if the edge is cut
+              else if (((ls_values[mask[0]] > iso_level) &&
```

Comment:
This is the `else if` to the `if` (line 6941) or `else if` (6940)? It feels odd that the indentation has changed.